python/lsst/ts/watcher/rules/atcamera_dewar.py

```diff
@@ -0,0 +1,420 @@
+                or a critical level (not usually both).
+            type: object
+            properties:
+                min_warning_ccd_temp:
+                    description: Minimum CCD temperature (C) below which warning.
+                    type: number
+                    default: -95
+                min_serious_ccd_temp:
+                    description: Minimum CCD temperature (C) below which serious.
+                    type: number
+                min_critical_ccd_temp:
+                    description: Minimum CCD temperature (C) below which critical.
+                    type: number
+                max_warning_ccd_temp:
+                    description: Maximum CCD temperature (C) above which warning.
+                    type: number
+                    default: -93
+                max_serious_ccd_temp:
+                    description: Maximum CCD temperature (C) above which serious.
+                    type: number
+                max_critical_ccd_temp:
+                    description: Maximum CCD temperature (C) above which critical.
+                    type: number
+                max_warning_cold_plate_temp:
+                    description: Maximum cold plate temperature (C) above which warning.
+                    type: number
+                    default: -110
+                max_serious_cold_plate_temp:
+                    description: Maximum cold plate temperature (C) above which serious.
+                    type: number
+                max_critical_cold_plate_temp:
+                    description: Maximum cold plate temperature (C) above which critical.
+                    type: number
+                max_warning_cryo_head_temp:
+                    description: Maximum cryo head temperature (C) above which warning.
+                    type: number
+                    default: -140
+                max_serious_cryo_head_temp:
+                    description: Maximum cryo head temperature (C) above which serious.
+                    type: number
+                max_critical_cryo_head_temp:
+                    description: Maximum cryo head temperature (C) above which critical.
+                    type: number
+                max_warning_vacuum:
+                    description: Maximum dewar vacuum (Torr) above which warning.
+                    type: number
+                    default: 1.0e-6
+                max_serious_vacuum:
+                    description: Maximum dewar vacuum (Torr) above which serious.
+                    type: number
+                max_critical_vacuum:
+                    description: Maximum dewar vacuum (Torr) above which critical.
+                    type: number
+                temperature_window:
+                    description: Period of time (seconds) over which to median temperatures.
+                    type: number
+                    default: 180
+                    exclusiveMinimum: 0
+                temperature_hysteresis:
```

Comment:
That's fair. I added the following to the description of each hysteresis:
```
The amount by which a value must improve past an
alarm threshold before the alarm severity decreases.
```
To put it another way: the thresholds set when an alarm goes off, the hysteresis determines when it goes off again. This is to avoid small fluctuations right at a level from making an alarm stutter on and off.

Reply:
That's great. Thanks for the explanation.
Hysteresis feels like the incorrect term however. 
Maybe  "improvement_condition", "delta" or "severity_delta" depending on the length.
You're call. I'm not sure there is an easy answer here.

Reply:
I am already using "hysteresis" to mean exactly this for all ESS-based rules and in the ThresholdHandler, so it'll be a big change. It's one I'm willing to consider, but I'd like to do it on a separate ticket because it involves many existing rules.